src/eia930.py

```diff
@@ -441,12 +441,19 @@ def manual_930_adjust(raw: pd.DataFrame):
     ovec_col = get_int_columns("PJM", raw.columns, ["OVEC"])
     raw.loc[:, ovec_col] = raw.loc[:, ovec_col] * -1
 
-    # Interchange AZPS - SRP is wonky. Use SRP - AZPS (inverted)
+    # Interchange AZPS - SRP is wonky before 6/1/2020 7:00 UTC. Use SRP - AZPS (inverted)
     azps_srp = get_int_columns("AZPS", raw.columns, ["SRP"])
     srp_azps = get_int_columns("SRP", raw.columns, ["AZPS"])
-    raw.loc[:, azps_srp] = (raw.loc[:, srp_azps] * (-1)).rename(
+    replacement = (raw.loc[:, srp_azps] * (-1)).rename(
         columns={srp_azps[0]: azps_srp[0]}  # rename so Pandas will do the right thing
     )
+    raw.loc[:"2020-06-01T07:00+00", azps_srp] = replacement[:"2020-06-01T07:00+00"]
+    # Update total interchange
+    all_cols = [c for c in get_int_columns("AZPS", raw.columns) if "ALL" not in c]
```

Comment:
This should exclude the total interchange column from this list, correct? This will only return all of the BA to BA interchange columns?

Reply:
yep, that's done by the `if "ALL" not in c` in the list comprehension. not the most readable but it is correct